src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicSurfaceClientTransformer.java

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer.csharp;
+
+import com.google.api.codegen.ApiConfig;
+import com.google.api.codegen.gapic.GapicCodePathMapper;
+import com.google.api.codegen.transformer.SurfaceTransformerContext;
+import java.io.File;
+
+public class CSharpGapicSurfaceClientTransformer extends CSharpGapicSurfaceCommonTransformer {
```

Comment:
I would very strongly prefer the existing CSharpGapicSurfaceTransformer to be parameterized over making subclasses of a common variant.